wpa_supplicant/ctrl_iface.c

```diff
@@ -924,6 +924,17 @@ static int wpa_supplicant_ctrl_iface_set(struct wpa_supplicant *wpa_s,
 #endif /* CONFIG_WNM */
 	} else if (os_strcasecmp(cmd, "enable_dscp_policy_capa") == 0) {
 		wpa_s->enable_dscp_policy_capa = !!atoi(value);
+#ifdef CONFIG_SAE
+	} else if (os_strcasecmp(cmd, "sae_pwe") == 0) {
+		wpa_s->conf->sae_pwe = atoi(value);
+		if (wpa_s->conf->sae_pwe < 0 ||
+		    wpa_s->conf->sae_pwe > 2) {
```

Comment:
Shouldn't the validity be checked before setting the parameter in wpa_s?

Reply:
Thanks, fixed it.